scripts/get_buffalo.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+BUFFALO_URL="https://github.com/gobuffalo/buffalo/releases/download/v0.12.3/buffalo_0.12.3_linux_amd64.tar.gz"
+TAR_GZ="buffalo_0.12.3_linux_amd64.tar.gz"
+
+curl -L -o ${TAR_GZ} ${BUFFALO_URL}
+tar -xvf ${TAR_GZ}
+mv buffalo-no-sqlite /usr/local/bin/buffalo
```

Comment:
dont you need `+x`

Reply:
I think it should already be executable in the tarball that gets downloaded, but no harm in doing so!